WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -585,15 +588,26 @@ private extension AuthenticationManager {
             // Tries re-syncing to get an updated store list,
             // then attempts to present epilogue again.
             ServiceLocator.stores.synchronizeEntities { [weak self] in
-                self?.presentLoginEpilogue(in: navigationController, for: credentials, onDismiss: onDismiss)
+                self?.presentLoginEpilogue(in: navigationController, for: credentials, source: nil, onDismiss: onDismiss)
             }
         })
         let installJetpackUI = ULErrorViewController(viewModel: viewModel)
         navigationController.show(installJetpackUI, sender: nil)
     }
 
-    func startStorePicker(with siteID: Int64? = nil, in navigationController: UINavigationController, onDismiss: @escaping () -> Void = {}) {
-        storePickerCoordinator = StorePickerCoordinator(navigationController, config: .login)
+    func startStorePicker(with siteID: Int64? = nil,
+                          source: SignInSource?,
```

Comment:
Nit: Would it make sense to assign a `nil` default value for this parameter? (To avoid sending `nil` from the calling locations)

Reply:
sure, updated it in https://github.com/woocommerce/woocommerce-ios/pull/7964/commits/2dfc80732927942900f67d7f697af0206d2ba924